docs-v2/integrations/all/1password-scim.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: 1Password SCIM
+sidebarTitle: 1Password SCIM
+---
+
+import Overview from "/snippets/overview.mdx"
+import PreBuiltTooling from "/snippets/generated/1password-scim/PreBuiltTooling.mdx"
+import PreBuiltUseCases from "/snippets/generated/1password-scim/PreBuiltUseCases.mdx"
+
+<Overview />
+<PreBuiltTooling />
+<PreBuiltUseCases />
+
+## Access requirements
+| Pre-Requisites | Status | Comment |
+| - | - | - |
+| Paid dev account | ✅ | 1Password SCIM is available on Business plans. |
+| Paid test account | ❓ |  |
+| Partnership | ❓ | |
+| App review | ❓ |  |
+| Security audit | ❓ | |
+
+## Setup guide
+
+_No setup guide yet._
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+<Note>Contribute improvements to the setup guide by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/1password.mdx)</Note>
+
+## Useful links
+
+- [1Password SCIM API documentation](https://support.1password.com/scim/)
+- [1Password Admin Console](https://my.1password.com)
+- [1Password Authentication](https://support.1password.com/scim/)
+- [1Password SCIM Bridge Explained](https://blog.1password.com/1password-scim-bridge-explained/)
+- [1Password SCIM Bridge Troubleshooting](https://support.1password.com/scim-troubleshooting/)
+
+
+<Note>Contribute useful links by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/1password-scim.mdx)</Note>
+
+## API gotchas
+
+- 1Password SCIM uses API_KEY authentication with `Authorization: Bearer API_KEY` in the request header.
+- Nango supports automatic retries if 1Password SCIM API rate limits requests.
```

Comment:
Why is this a gotcha? We don't parse the headers of 1Password so I don't see any reason to include this specifically?